en/mozillavpn.xliff

```diff
@@ -0,0 +1,556 @@
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/views/ViewServers.qml</context><context context-type="linenumber">16</context></context-group>
+      <note annotates="source" from="developer">Select the Location of the VPN server</note>
+    </trans-unit>
+    <trans-unit id="vpn.servers.currentLocation">
+      <source xml:space="preserve">current location - %1</source>
+      <context-group purpose="location"><context context-type="linenumber">18</context></context-group>
+      <note annotates="source" from="developer">Accessibility description for current location of the VPN server</note>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/components/VPNControllerView.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.controller.deactivated">
+      <source xml:space="preserve">VPN is off</source>
+      <context-group purpose="location"><context context-type="linenumber">50</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">97</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">346</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">394</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.controller.activationSloagan">
+      <source xml:space="preserve">Turn on to protect your privacy</source>
+      <context-group purpose="location"><context context-type="linenumber">57</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">103</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">353</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">401</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.controller.connecting">
+      <source xml:space="preserve">Connecting</source>
+      <context-group purpose="location"><context context-type="linenumber">145</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.controller.activating">
+      <source xml:space="preserve">Masking connection and location</source>
+      <context-group purpose="location"><context context-type="linenumber">152</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.controller.activated">
+      <source xml:space="preserve">VPN is on</source>
+      <context-group purpose="location"><context context-type="linenumber">199</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.controller.active">
+      <source xml:space="preserve">Secure and private</source>
+      <context-group purpose="location"><context context-type="linenumber">206</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.controller.disconnecting">
+      <source xml:space="preserve">Disconnecting&#8230;</source>
```

Comment:
This is still open from the last PR: can we replace these HTML entities with the actual characters? Is this a byproduct of the export?